java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -379,6 +379,29 @@ public byte getByte(long index) {
     return offHeap.data.getByte(index * type.sizeInBytes);
   }
 
+  /**
+   * Get the byte list at index.
+   */
+  public byte[] getByteList(long index) {
+    assert type == DType.LIST;
+    assertsForGet(index);
+
+    int start = offHeap.offsets.getInt(index * DType.INT32.getSizeInBytes());
+    int end = offHeap.offsets.getInt((index + 1) * DType.INT32.getSizeInBytes());
+    int size = end-start;
```

Comment:
There are string APIs for this, that I think we should make generic to work for all lists. Probably add new methods for lists and have the string ones call into the list ones for backwards compatibility.